code/game/objects/structures/crates_lockers/largecrate_vr.dm

```diff
@@ -106,4 +106,14 @@
 		playsound(src, 'sound/items/poster_ripped.ogg', 50, 1)
 		icon_state = "otiecrate"
 		taped = 0
-	..()
\ No newline at end of file
+	..()
+
+/obj/structure/largecrate/animal/mlem
+	name = "Catgirl Crate"
+	desc = "A sketchy looking crate that seems to have had most marks and stickers removed. You can almost make out 'genetically-engineered subject'."
+	held_type = /mob/living/simple_animal/catgirl
+
+/obj/structure/largecrate/animal/awoo
```

Comment:
These should be named properly. Not mlem or awoo. Catgirl or Wolfgirl. Or awoo I guess because the file path is awoo.